.github/workflows/test.yml

```diff
@@ -171,6 +171,12 @@ jobs:
         profile: ["docker", "singularity", "conda"]
         exclude:
           - tags: "nf-test"
+          - profile: "conda"
+            tags: gbcms
+          - profile: "conda"
+            tags: polysolver
+          - profile: "conda"
+            tags: mutect1
```

Comment:
These aren't configured for pytest so we can remove this added block. This section of the tes.yml is related to legacy pytest-ing.